assets/js/WfParameter.js

```diff
@@ -277,11 +277,48 @@ export default class WfParameter extends React.Component {
     }
   }
 
+  displayConditionalUI(condition, type) {
+    // Checks if conditional UI should be displayed
```

Comment:
Could use a slightly more detailed comment: checks if the menu item referenced by this parameter is set to a value where we are visible